### DISCUSSION TITLE
未知原因导致消息会话头像昵称均被打上马赛克

### DISCUSSION THREAD

**Discussion by Emotion04** [cinit/QAuxiliary#1005] [Q&A] [answered]
如题。排查了我能想到的相关功能都没有解决，除非关掉qa

![S40604-14003968_com.tencent.mobileqq.png](https://github.com/cinit/QAuxiliary/assets/107828821/903335e6-d9ab-48d8-bd12-0dd3403965a9)

**Comment by HdShare** [comment, marked as answer]
> 如题。排查了我能想到的相关功能都没有解决，除非关掉qa
> 
> ![S40604-14003968_com.tencent.mobileqq.png](https://github.com/cinit/QAuxiliary/assets/107828821/903335e6-d9ab-48d8-bd12-0dd3403965a9)
> 
> 

娱乐功能-昵称/群名片打码
